soccer/gameplay/root_play.py

```diff
@@ -39,10 +39,18 @@ def execute_running(self):
         ################################################################################
 
         if main.game_state().is_stopped():
-            if not isinstance(self.play, plays.stopped.Stopped):
-                logging.info("Running 'Stopped' play due to game state change")
-                self.play = plays.stopped.Stopped()
-                self._currently_restarting = True
+            if main.game_state().is_placement():
```

Comment:
Is placement a "sub state" of stopped?  If so, be sure the referee module sets the GameState to stopped when there's a placement command.  If not, then this shouldn't go inside the "if stopped" above.